src/main/java/org/prebid/cache/handlers/cache/PostCacheHandler.java

```diff
@@ -47,6 +50,7 @@ public class PostCacheHandler extends CacheHandler {
 
     private static final String UUID_KEY = "uuid";
     private static final String SECONDARY_CACHE_KEY = "secondaryCache";
+    private static final String MODULE_STORAGE_PREFIX = "module.";
```

Comment:
so saving into the module storage is restricted here by the `module` prefix, but at the same time there isn't any guarantee that the module storage saves everything with that prefix (it could be any), so what's the matter of this check? Shouldn't we save everything into the module storage with that prefix to guarantee this check makes sense?

Am I missing something?